app/services/notifications/reactions/send.rb

```diff
@@ -58,7 +58,7 @@ def call
           notification = Notification.find_or_initialize_by(notification_params)
 
           old_json_data = notification.json_data
-          previous_siblings_size = notification.json_data["reaction"]["aggregated_siblings"].size if old_json_data
+          previous_siblings_size = old_json_data.dig("reaction", "aggregated_siblings")&.size if old_json_data
```

Comment:
I feel like I checked on this once- there are comment reaction notifications `where action = 'Reaction'`, but the json_data is for "comment" rather than "reaction", so the assumed structure isn't the same, there isn't a reaction key or aggregated siblings.

Not sure if that's the issue you're fixing.

Reply:
that is indeed what I'm trying to fix

Reply:
Thanks for the breakdown. I also did not have enough context and the frequency of the error didn't warrant a deeper solution.